spec/controllers/books_controller_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+describe BooksController do
+  describe 'GET #index' do
+    subject(:http_request) do
```

Comment:
There may be many http requests. Use a more descriptive name, such as `index_request`. Also, you can do this in just one line with curly brackets (`{}`).